### DISCUSSION TITLE
Unexpected High Contrast on non-HDR screen in Microsoft Edge while using Dual Screen

### DISCUSSION THREAD

**Discussion by String10** [String10/KGarden#41] [Issues]
I've been suffer a problem for a while: when I connect another screen to the laptop, sometimes the Edge window on the laptop screen will display like this:

![screenshot from Github with high contrast](https://github.com/String10/KGarden/assets/69832835/b51875be-4ccb-4772-b615-e71033ab5d97)

I don't know how to describe that problem before until I went into this post from reddit: [r/MicrosoftEdge: Dual screen HDR problem after update](https://www.reddit.com/r/MicrosoftEdge/comments/vla3ih/dual_screen_hdr_problem_after_update/). The edge window on the origin screen gets a high contrast.

Then I went to `edge://flags` as that post instructs and set `Force color profile` to `scRGB linear (HDR where available)`.

![edge settings: force color profile](https://github.com/String10/KGarden/assets/69832835/e843d343-ff57-4152-b50d-10d043595c21)

PS:
```
Microsoft Edge: 122.0.2365.66 (Official build) (64-bit)
Revision: ab363614b8f04700fdbbc3d9d7f9deca8102a1c4
Chrominum Version: 122.0.6261.95
OS: Windows 11 Version 22H2 (Build 22621.3155)
GPU: NVIDIA Geforce RTX 4060 Laptop GPU
```

**Comment by tale80** [comment]
Thanks, I had the same problem with the last Edge update on Fedora.
I have applied your solution and it's ok now !
